pyproject.toml

```diff
@@ -1,3 +1,96 @@
+[project]
```

Comment:
If I understand correctly, we'll also need a block like this:
```
[tool.hatch.build.targets.wheel]
packages = ["src/scriptworker"]
```

Reply:
Weird. I tried a similar pyproject.toml without a `packages` block, and it failed to even make a package for me. (We also have this block in ship it: https://github.com/mozilla-releng/shipit/blob/0e50df631305a2002e934d9db81dfb762a55c99a/api/pyproject.toml#L73-L74.)

Presumably there's some magic under the hood here automatically adding `src`?